flang/test/Fir/commute.fir

```diff
@@ -1,4 +1,4 @@
-// RUN: fir-opt %s | tco | FileCheck %s
+// RUN: fir-opt --basic-cse %s | tco | FileCheck %s
```

Comment:
Since `tco` is already in the pipeline, what does this do?

Reply:
CSE for commutative ops is not in the MLIR CSE yet. A patch is up for review here: https://reviews.llvm.org/D123492